importer/tasks.py

```diff
@@ -0,0 +1,11 @@
+"""
```

Comment:
No from __future__ import unicode here? It may be a good idea to add verifying this exists to our list of things to check when tox runs. 

We don't need it here but it would be good to be consistent across the application and to prevent possible oversights for future commits